src/main/java/net/mcreator/element/parts/VillagerTradeEntry.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.parts;
+
+import net.mcreator.generator.mapping.MappableElement;
+import net.mcreator.generator.mapping.NameMapper;
+import net.mcreator.minecraft.DataListEntry;
+import net.mcreator.workspace.Workspace;
+
+import javax.annotation.Nonnull;
+
+public class VillagerTradeEntry extends MappableElement {
```

Comment:
This is VillagerProffesion, not VillagerTradeEntry

Reply:
?? I'm afraid I don't understand...

Reply:
What about when I will create the "Villager Profession" mod element? There can't be 2 "VillagerProfession"

Reply:
There already are 2 Procedures and they feel absolutely fine...

Reply:
? There is only 1 file named `Procedure` in `mcreator/element/parts`

Reply:
Yes, I thought you should rename VillagerTradeEntry to VillagerProffesion as this object holds villager profession type

Reply:
It's done!